docs/deployment/README.md

```diff
@@ -147,292 +127,288 @@ These resources only need to be created once per account.
+appropriate values.
+
+**terraform.tfvars:**
+
+```hcl
+region = "us-east-1"
+
+# Layer ARN deployed in the previous step
+cumulus_message_adapter_lambda_layer_arn = "arn:aws:lambda:us-east-1:12345:layer:Cumulus_Message_Adapter:4"
+
+# URS users you want to have access to your API
+archive_api_users = [ 'some_user' ]
+
+permissions_boundary_arn = "arn:aws:iam::12345:policy/NGAPShNonProdRoleBoundary"
+prefix                   = "PREFIX"
+buckets = {
+  internal = {
+    name = "PREFIX-internal"
+    type = "internal"
+  }
+  private = {
+    name = "PREFIX-private"
+    type = "private"
+  },
+  protected = {
+    name = "PREFIX-protected"
+    type = "protected"
+  },
+  public = {
+    name = "PREFIX-public"
+    type = "public"
+  }
+}
+subnet_ids    = ["subnet-12345"]
+system_bucket = "PREFIX-internal"
+vpc_id        = "vpc-12345"
+key_name      = "MY-KEY"
+
+cmr_client_id   = "cumulus-core-PREFIX"
+cmr_environment = "UAT"
+cmr_password    = "password"
+cmr_provider    = "CUMULUS"
+cmr_username    = "username"
+
+urs_client_id       = "asdf"
+urs_client_password = "password"
+
+token_secret = "asdf"
+
+data_persistence_remote_state_config = {
+  bucket = "PREFIX-tf-state"
+  key    = "PREFIX/data-persistence/terraform.tfstate"
+  region = "us-east-1"
+}
+
+# Optional
+oauth_provider = "earthdata"    # Can also be set to "launchpad"
 ```
 
-__Note:__ Be sure to copy the urls, as you will use them to update your EarthData application.
+**Note:** The `token_secret` is a string value used for signing and verifying [JSON Web Tokens (JWTs)](https://jwt.io/) issued by the API. For security purposes, it is **strongly recommended that this value be a 32-character string**.
+
+**Note:** The `data_persistence_remote_state_config` section should contain the
+remote state values that you configured in
+`data-persistence-tf/terraform.tf`. These settings allow `cumulus-tf` to
+determine the names of the resources created in `data-persistence-tf`.
+
+**Note:** When deploying to NGAP, the `permissions_boundary_arn` should refer to
+`NGAPShNonProdRoleBoundary`.
+
+Run `terraform init` if this is your first time deploying this module.
 
-### Update Earthdata Application.
+Run `terraform apply` to deploy the resources. Type `yes` when prompted to confirm that you want to create the resources. Assuming the operation is succesful, you should see output like this:
+
+```shell
+Apply complete! Resources: 292 added, 0 changed, 0 destroyed.
+
+Outputs:
+
+archive_api_redirect_uri = https://abc123.execute-api.us-east-1.amazonaws.com/dev/token
+archive_api_uri = https://abc123.execute-api.us-east-1.amazonaws.com/dev/
+distribution_redirect_uri = https://abc123.execute-api.us-east-1.amazonaws.com/DEV/login
+distribution_url = https://abc123.execute-api.us-east-1.amazonaws.com/DEV/
+```
 
-You will need to add two redirect urls to your EarthData login application.
+__Note:__ Be sure to copy the redirect URLs, as you will use them to update your EarthData application.
+
```

Comment:
If we retain the repository information above, it might be good to include a reminder to not commit the secrets file.  Seems obvious....but....

Reply:
4468a5e